frontend/scripts/react-components/tool-selector/tool-selector.component.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Heading from 'react-components/shared/heading';
+import GridListItem from 'react-components/shared/grid-list-item/grid-list-item.component';
+import PropTypes from 'prop-types';
+import TopCards from 'react-components/tool-selector/top-cards';
+
+import 'react-components/tool-selector/tool-selector.scss';
+
+function ToolSelector({ items, step, setCommodity, setCountry, commodityName, countryName }) {
+  const renderTitle = () => {
+    const titleParts = ['commodity', 'sourcing country', 'supply chain'];
+    return (
+      <Heading size="lg" align="center" className="tool-selector-title">
+        {step + 1}. Choose one {titleParts[step]}
+      </Heading>
+    );
+  };
+  const setItemFunction = step === 0 ? setCommodity : setCountry;
+  return (
+    <div className="c-tool-selector">
+      <div className="row columns">{renderTitle()}</div>
+      <div className="row columns">
+        <div className="grid-list">
```

Comment:
This might confusing in the future, could we use a different className, and use grid-list only in the grid-list component?